osgar/tools/lidarview.py

```diff
@@ -280,6 +305,12 @@ def lidarview(gen, caption_filename, callback=False):
             screen.blit(foreground, (0, 0))
             pygame.display.flip() 
 
+            if out_video is not None:
+                # hacked image conversion - there is surely direct way!
+                pygame.image.save(screen, "tmp.jpg")
+                im = cv2.imread("tmp.jpg", 1)
+                writer.write(im)
```

Comment:
https://stackoverflow.com/a/53108946/3185929

Reply:
Would it be possible to do this in memory directly as the above SO link suggests?

Reply:
fixed, thanks for reminder & sorry (I forgot about mix of "create video" commit)